packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -2113,7 +2120,7 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         transactionContext.ovmL1TXORIGIN = _transaction.l1TxOrigin;
         transactionContext.ovmGASLIMIT = gasMeterConfig.maxGasPerQueuePerEpoch;
 
-        messageRecord.nuisanceGasLeft = _getNuisanceGasLimit(_transaction.gasLimit);
+        messageRecord.nuisanceGasLeft = _transaction.gasLimit;
```

Comment:
`_getNuisanceGasLimit` did this: 

```
        return _gasLimit < gasleft() ? _gasLimit : gasleft();
```

Should the initial `nuisanceGasLeft` value not also be bounded by `gasleft()`?

Reply:
We're going to shift this to the StateTransitioner, so I think it's fine to do this way during initialization.